aries_cloudagent/connections/base_manager.py

```diff
@@ -4,6 +4,9 @@
 For Connection, DIDExchange and OutOfBand Manager.
 """
 
+import logging
+from typing import Sequence, Tuple
+
 from ..core.profile import ProfileSession
```

Comment:
I'll clean this up later but just FYI, it should be ordered by (reverse) proximity.. ie. `..core` and `..wallet` come before `.models`, and from there things should be ordered alphabetically.

Reply:
Thanks, I will keep this in mind from now on.